tests/login_page.py

```diff
@@ -18,7 +20,7 @@ def setup(self):
         driver = webdriver.Chrome(
             executable_path="/home/ihor/PycharmProjects/pythonQALightSelenium/drivers/chromedriver")
         driver.maximize_window()
-        driver.get("https://qa-complex-app-for-testing.herokuapp.com/")
+        driver.get(start_page_locators.START_PAGE_URLS)
         driver.implicitly_wait(time_to_wait=20)  # ждем пока найдет элемент
         yield driver
```

Comment:
Please return start_page instead of driver. Like I do https://github.com/qalight-python-automation/main-qalight-qacomplexapp-tests/blob/1626c714271f4eef35361c6dd5c6bd0b4a1cf977/tests/test_start_page.py#L23